src/nrtk_explorer/library/nrtk_transforms.py

```diff
@@ -12,7 +13,13 @@
 logger.setLevel(logging.INFO)
 
 try:
-    from pybsm.otf import darkCurrentFromDensity
+    import pybsm
+
+    if Version(pybsm.__version__) < Version("0.5"):
```

Comment:
if we put constraint on pybsm version we could clean that code. Do we need to support `<0.5`?